lib/core.fs

```diff
@@ -406,6 +406,25 @@ ps-drop,
 ret,
 end-code
 
+( -- addr )
+code here
+ps-dup,
+$C000 ]* A ld, A H ld,
```

Comment:
The order in memory of the register is usually the opposite, `L` first, then `H`, that will make easier to interop with Forth.

Reply:
That makes sense, but it is implemented this way around for `!` and `@` at the moment, so I’ll make sure change them everywhere.